packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -277,6 +281,10 @@ export default async function start(basedir, argv) {
     }
   }
 
+  const { wallet, defaultManagerType } = JSON.parse(
```

Comment:
I don't know how this `options.json` works.. I want to make sure we aren't encouraging validator operators to think they can switch the type after creating their node. I'm a little suprised to think there are multiple places to set this value.

Reply:
`options.json` was the only path I could see between `make scenario3-setup` and `make scenario3-run-client`; i.e. between `ag-solo init` and `ag-solo start`.

Yes, it's not pretty, but I figure when in Rome...

Reply:
@michaelfig does this seem good?